Formula/mongodb-csfle.rb

```diff
@@ -0,0 +1,23 @@
+class MongodbCsfle < Formula
+  desc "MongoDB csfle shared library for Client Side Encryption"
+  homepage "https://www.mongodb.com/"
+
+  # TODO(MONGOSH-1193): Bump to production version after 6.0 is released
+  url "https://downloads.mongodb.com/osx/mongo_csfle_v1-macos-x86_64-enterprise-6.0.0-rc0.tgz"
```

Comment:
If we merge this, are we effectively releasing the rc0 csfle library into the wild? Is that something we actually want to do? Or should we be waiting for GA?

Reply:
It would be really nice to have this available for testing for now. I’m not super worried about this, since it is still something new that isn’t even really announced or documented yet – the idea here (as indicated by the TODO comment) would be to only switch to GA versions once those are available. (The 5.3.x versions are buggy in a way that makes them effectively unusable, at least for us.)